testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/system/system_capabilities.t.js

```diff
@@ -0,0 +1,120 @@
+ */
+StartTest(function(t) {
+  var waitForStoreToLoad = {
+    waitFor: 'storesToLoad', args: function() {
+        return t.cq1('nx-coreui-capability-list').getStore();
+    }
+  },
+      // Name of Capability tested
+      name = 'Base URL: Force';
+
+  t.describe('Given an admin when the admin is administering Capabilities', function(t) {
+    t.it('then CRUD should work', function(t) {
+      t.chain(
+        t.openPageAsAdmin('admin/system/capabilities'),
+        // CREATE: Wait for create capability button to be enabled
+        {waitFor: 'CQ', args: 'nx-coreui-capability-list button[action=new][disabled=false]'},
+
+        // Press the create capability button
+        {click: '>>nx-coreui-capability-list button[action=new]'},
+        // and wait for list
+        function(next) {
+          t.waitForAnimations(next);
+        },
+        // Find the capability and click it
+        function(next) {
+          var grid = t.cq1('nx-coreui-capability-selecttype'),
+              baseURLRow = t.getRow(grid, grid.getStore().find('name', name));
+
+          t.click(baseURLRow, next);
+        },
+        // and wait for details to load
+        function(next) {
+          t.waitForAnimations(next);
+        },
+        // Press the "Add" button and wait for grid to refresh
+        Ext.apply(Ext.clone(waitForStoreToLoad),
+          {trigger: {click: '>>nx-coreui-capability-add button[action=add]'}}
+        ),
+        // READ: Verify that
+        function(next) {
+          var grid = t.cq1('nx-coreui-capability-list'),
+              store = grid.getStore(),
+              model = store.findRecord('typeName', name);
+
+          // we can find the created capability
+          t.ok(model, 'Capability ' + name + ' exists');
+          // and values correspond
+          t.is(model.get('typeId'), 'baseurl.force');
+
+          // Back to the detail panel view
+          t.click(t.getRow(grid, grid.getView().indexOf(model)));
+
+          next();
+        },
+        //and wait for view to load
+        function(next) {
+          t.waitForAnimations(next);
+        },
+
+        // UPDATE: Select Settings pill and wait for transition
+        function(next) {
+          t.waitForAnimations(function() {
+            t.click('>>nx-drilldown-details tab[text=Settings]');
+            next();
+          });
+        },
+        // then set capability to disabled
+        {click: '>>nx-coreui-capability-settings field[name=enabled]'},
+        // Save changes
+        {click: '>>nx-coreui-capability-settings button[action=save]'},
+
+        // Verify that
+        function(next) {
+          var grid = t.cq1('nx-coreui-capability-list'),
+              store = grid.getStore(),
+              model = store.findRecord('typeName', name);
+
+          // we can find the updated record in store
+          t.ok(model, 'Capability ' + name + ' exists');
+          // and is it disabled
+          t.is(model.get('enabled'), 0);
```

Comment:
This assertion is currently failing on CI: http://bamboo.s/browse/NX3-OSSF520-TEST-1/test/case/153647346